marketing/templates/email_report.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <p>Hello from OCIM,</p>
+    {% if sent_emails %}
```

Comment:
Since we are [checking](https://github.com/open-craft/opencraft/pull/789/files#diff-780cc6cdf2129833893b838950d6686891522ce2a81b4aa51c612e5be5c81c1aR121) if `sent_emails` exists - this might not required